src/main/webapp/app/entities/metis/post.model.ts

```diff
@@ -19,6 +19,7 @@ export class Post extends Posting {
     public conversation?: Conversation;
     public courseWideContext?: CourseWideContext;
     public displayPriority?: DisplayPriority;
+    public resolved?: boolean;
```

Comment:
It would be useful for consistency to also have the other two new properties added to the client model. `vote_count` and `answer_count`. You could for example use them already in the `post-reactions-bar.component.html`.

Reply:
Thanks!